include/alpaka/acc/AccCpuOmp2Blocks.hpp

```diff
@@ -116,6 +116,18 @@ namespace alpaka
             using type = AccCpuOmp2Blocks<TDim, TIdx>;
         };
 
+        //! The CPU OpenMP 2.0 block single thread accelerator type trait specialization.
+        template<typename TDim, typename TIdx>
+        struct IsSingleThreadAccelerator<AccCpuOmp2Blocks<TDim, TIdx>> : std::true_type
```

Comment:
Could you please rename the traits to `isMultiThreadAcc` to follow the typical naming of alpaka.

Reply:
Will do.

Reply:
Done